packages/eslint-config-ckeditor5/eslint.config.mjs

```diff
@@ -0,0 +1,491 @@
+				objectLiteralTypeAssertions: 'allow-as-parameter'
+			} ],
+
+			'@typescript-eslint/consistent-type-imports': 'error',
+
+			'@typescript-eslint/explicit-module-boundary-types': [ 'error', {
+				allowedNames: METHODS_THAT_USE_AS_CONST_INSTEAD_OF_RETURN_TYPE,
+				allowArgumentsExplicitlyTypedAsAny: true
+			} ],
+
+			'@typescript-eslint/explicit-member-accessibility': [ 'error', {
+				accessibility: 'explicit',
+				overrides: {
+					constructors: 'off'
+				}
+			} ],
+
+			'@typescript-eslint/no-confusing-non-null-assertion': 'error',
+
+			'@typescript-eslint/no-empty-function': 'off',
+
+			'@typescript-eslint/no-empty-interface': 'off',
+
+			'@typescript-eslint/no-explicit-any': 'off',
+
+			'@typescript-eslint/no-inferrable-types': 'off',
+
+			'@typescript-eslint/no-invalid-void-type': 'error',
+
+			'@typescript-eslint/no-non-null-asserted-nullish-coalescing': 'error',
+
+			'@typescript-eslint/no-non-null-assertion': 'off',
+
+			'@typescript-eslint/parameter-properties': 'error',
+
+			'@typescript-eslint/unified-signatures': 'error',
+
+			/*
+			╭─────────────────╮
+			│ Extension rules │
+			╰─────────────────╯
+			*/
+
+			'no-unused-expressions': 'off',
+			'@typescript-eslint/no-unused-expressions': 'error',
+
+			'no-unused-vars': 'off',
+			'@typescript-eslint/no-unused-vars': 'error',
+
+			'no-use-before-define': 'off',
+			'@typescript-eslint/no-use-before-define': [ 'error', {
+				functions: false,
+				classes: false,
+				variables: true,
+				typedefs: false,
+				ignoreTypeReferences: true
+			} ],
+
+			'no-array-constructor': 'off',
+			'@typescript-eslint/no-array-constructor': 'error',
+
+			'no-useless-constructor': 'off',
+			'@typescript-eslint/no-useless-constructor': 'error',
+
+			/*
+			╭───────────────────────────────────────────────────────╮
+			│ Stylistic rules for source code layout and formatting │
+			╰───────────────────────────────────────────────────────╯
+			*/
+
+			'@stylistic/member-delimiter-style': 'error',
+
+			'@stylistic/type-annotation-spacing': 'error'
+		}
+	}
+];
+
+const rulesSourceCode = [
+	{
+		plugins: {
+			'ckeditor5-rules': ckeditor5Rules
+		},
+
+		files: [ '**/src/**/*.@(ts|tsx)' ],
+
+		settings: {
+			disallowedCrossImportsPackages: [
+				'ckeditor5-watchdog'
+			]
+		},
+
+		rules: {
+			'ckeditor5-rules/no-relative-imports': 'error',
+
+			'ckeditor5-rules/ckeditor-error-message': 'error',
+
+			'ckeditor5-rules/no-cross-package-imports': 'error',
+
+			'ckeditor5-rules/no-scoped-imports-within-package': 'error',
+
+			'ckeditor5-rules/use-require-for-debug-mode-imports': 'error',
+
+			'ckeditor5-rules/no-istanbul-in-debug-code': 'error',
+
+			'ckeditor5-rules/allow-declare-module-only-in-augmentation-file': 'error',
+
+			'ckeditor5-rules/allow-imports-only-from-main-package-entry-point': 'error',
+
+			'ckeditor5-rules/require-as-const-returns-in-methods': [ 'error', {
+				methodNames: METHODS_THAT_USE_AS_CONST_INSTEAD_OF_RETURN_TYPE
+			} ],
```

Comment:
```suggestion
			'ckeditor5-rules/require-as-const-returns-in-methods': [ 'error', {
				methodNames: [ 'requires', 'pluginName' ]
			} ],
```

To unify. We have more rules that require configuration, but this one uses `const` only.

Reply:
Done in https://github.com/ckeditor/ckeditor5-linters-config/pull/60/commits/d26bb19ec47679927d8917d68eb2f05d9d46c5e3.